test/block/blockupload.test.js

```diff
@@ -67,18 +67,14 @@ describe('Block Upload', function() {
             uploadFiles: [{
                 fileUrl: targetUrl,
                 filePath: testFile,
-                fileSize: 15,
-                multipartHeaders: { partHeader: 'test' },
-                minPartSize: 10,
-                maxPartSize: 25
+                fileSize: 15
             }],
             headers: {
                 // Asset Compute passes through content-type header
                 'content-type': 'image/jpeg',
             },
             concurrent: true,
             maxConcurrent: 5,
-            preferredPartSize: 7
```

Comment:
test was testing a non-multipart upload test, so I removed these items.

the test after this tests multipart